src/protocol/frsky2way_cc2500.c

```diff
@@ -297,7 +297,14 @@ static void frsky2way_parse_telem(u8 *pkt, int len)
     Telemetry.value[TELEM_FRSKY_RSSI] = pkt[5]; 	// Value in Db
     TELEMETRY_SetUpdated(TELEM_FRSKY_RSSI);
 
+    Telemetry.value[TELEM_FRSKY_LQI] = pkt[len-1] & 0x7f;
+    TELEMETRY_SetUpdated(TELEM_FRSKY_LQI);
+
+    Telemetry.value[TELEM_FRSKY_LRSSI] = (s8)pkt[len-2] / 2 - 70; 	// Value in Db
```

Comment:
Would it be better to also convert the regular RSSI value to (approximate) dBm?  Some people may already have alarms set for current values.